lib/model_configuration/attribute/validation/factory.rb

```diff
@@ -1,7 +1,13 @@
+require_relative '../validation'
+require_relative 'length'
+require_relative 'numericality'
+
 class ModelConfiguration::Attribute::Validation::Factory
 
   def build(attribute, key, args)
     case key
+    when "length"
+      ModelConfiguration::Attribute::Validation::Length.new(attribute, key, args)
     when "numericality"
       ModelConfiguration::Attribute::Validation::Numericality.new(attribute, key, args)
     else
```

Comment:
I really like the idea of breaking validation into small classes like `ModelConfiguration::Attribute::Validation::Whatever`

:+1: easier to test!!!